static/app/views/replays/detail/network/details/onboarding.tsx

```diff
@@ -1,16 +1,88 @@
 import styled from '@emotion/styled';
 
 import Alert from 'sentry/components/alert';
+import {Button} from 'sentry/components/button';
 import {CodeSnippet} from 'sentry/components/codeSnippet';
 import ExternalLink from 'sentry/components/links/externalLink';
+import {IconClose, IconInfo} from 'sentry/icons';
 import {t, tct} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
+import useDismissAlert from 'sentry/utils/useDismissAlert';
 import useOrganization from 'sentry/utils/useOrganization';
 import useProjectSdkNeedsUpdate from 'sentry/utils/useProjectSdkNeedsUpdate';
 import {Output} from 'sentry/views/replays/detail/network/details/getOutputType';
 import type {TabKey} from 'sentry/views/replays/detail/network/details/tabs';
 import type {NetworkSpan} from 'sentry/views/replays/types';
 
+export const useDismissReqRespBodiesAlert = () => {
+  const organization = useOrganization();
+  return useDismissAlert({
+    key: `${organization.id}:replay-network-bodies-alert-dismissed`,
+  });
+};
+
+export function ReqRespBodiesAlert({
+  isNetworkDetailsSetup,
+}: {
+  isNetworkDetailsSetup: boolean;
+}) {
+  const {dismiss, isDismissed} = useDismissReqRespBodiesAlert();
+
+  if (isDismissed) {
+    return null;
+  }
+
+  const message = isNetworkDetailsSetup
+    ? tct(
+        'Click on a [fetch] or [xhr] request to see request and response payloads. [link]',
+        {
+          fetch: <code>fetch</code>,
+          xhr: <code>xhr</code>,
+          link: (
+            <ExternalLink
+              href="https://github.com/getsentry/sentry-javascript/issues/7103"
```

Comment:
Hmm would this be the right link?

```suggestion
              href="https://docs.sentry.io/platforms/javascript/session-replay/configuration/#network-details"
```